.circleci/config.yml

```diff
@@ -36,13 +35,16 @@ jobs:
       - deploy:
           name: Push
           command: |
-            apk add --no-cache git
             # only push tagged commits on master branch (i.e., versions)
             if [ "${CIRCLE_BRANCH}" = "master" ] && [ -n "$(git tag --list --points-at HEAD)" ]; then
               set -x
               docker tag "${DOCKER_IMAGE}:${CIRCLE_SHA1}" "${DOCKER_IMAGE}:${CIRCLE_BRANCH}"
               docker push "${DOCKER_IMAGE}:${CIRCLE_BRANCH}"
             fi
+  deploy:
+    docker:
+      - image: alpine
+    steps:
       - deploy:
           name: Deploy to Staging
           command: |
```

Comment:
Is the `if [ "${CIRCLE_BRANCH}" = "master" ] && [ -n "$(git tag --list --points-at HEAD)" ]; then` below still needed?

Reply:
probably not but I'd rather be safe than sorry for a bit :)